src/pages/signup/signup.tsx

```diff
@@ -99,11 +101,22 @@ const Signup: React.FC = () => {
       team_leader: false,
       stepsByDate: [],
       admin: true,
+      step_goal: 0,
     });
   };
 
   // sign up with google //
   const googleAuth = async () => {
+    const currentDate: Date = new Date();
+    const adminRef = doc(FirestoreDB, 'admin', 'admin');
+    const adminSnap = await getDoc(adminRef);
```

Comment:
Instead of directly reading it from firebase, we should use admin context.
Michael is implementing admin context, so talk with him.